presto-hive/src/main/java/io/prestosql/plugin/hive/CreateEmptyPartitionProcedure.java

```diff
@@ -97,14 +96,14 @@ public void createEmptyPartition(ConnectorSession session, String schema, String
     private void doCreateEmptyPartition(ConnectorSession session, String schemaName, String tableName, List<Object> partitionColumnNames, List<Object> partitionValues)
     {
         TransactionalMetadata hiveMetadata = hiveMetadataFactory.get();
-        HiveTableHandle tableHandle = (HiveTableHandle)hiveMetadata.getTableHandle(session, new SchemaTableName(schemaName, tableName));
+        HiveTableHandle tableHandle = (HiveTableHandle) hiveMetadata.getTableHandle(session, new SchemaTableName(schemaName, tableName));
 
         List<String> actualPartitionColumnNames = tableHandle.getPartitionColumns().stream()
                 .map(HiveColumnHandle::getName)
                 .collect(toImmutableList());
 
         if (!Objects.equals(partitionColumnNames, actualPartitionColumnNames)) {
-            throw new PrestoException(INVALID_PROCEDURE_ARGUMENT, "input partition column names doesn't match actual partition column names");
+            throw new PrestoException(INVALID_PROCEDURE_ARGUMENT, "Provided partition column names don't match actual partition column names");
```

Comment:
We avoid contractions in messages for clarity (i'm aware previous code used them)

```suggestion
            throw new PrestoException(INVALID_PROCEDURE_ARGUMENT, "Provided partition column names do not match actual partition column names");
```